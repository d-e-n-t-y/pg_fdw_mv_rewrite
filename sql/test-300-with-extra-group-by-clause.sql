-- FIXME: re-grouping based on a strict superset of MV group by terms is not currently supported

explain (VERBOSE, COSTS OFF) select key, hidden, COUNT (value) from test group by key, hidden order by key;

select key, hidden, COUNT (value) from test group by key, hidden order by key;
