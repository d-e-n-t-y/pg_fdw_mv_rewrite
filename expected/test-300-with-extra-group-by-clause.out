-- FIXME: re-grouping based on a strict superset of MV group by terms is not currently supported
explain (VERBOSE, COSTS OFF) select key, hidden, COUNT (value) from test_lb group by key, hidden order by key;
                              QUERY PLAN                              
----------------------------------------------------------------------
 Sort
   Output: key, hidden, (count(value))
   Sort Key: test_lb.key
   ->  HashAggregate
         Output: key, hidden, count(value)
         Group Key: test_lb.key, test_lb.hidden
         ->  Foreign Scan on public.test_lb
               Output: key, hidden, value
               Remote SQL: SELECT key, hidden, value FROM public.test
(9 rows)

select key, hidden, COUNT (value) from test_lb group by key, hidden order by key;
 key  | hidden  |  count  
------+---------+---------
 key1 | hidden1 |      10
 key2 | hidden2 | 1000000
(2 rows)

